components/service/glean/docs/pings/custom.md

```diff
@@ -25,6 +29,14 @@ search:
   description: >
     A ping to record search data.
   include_client_id: false
+  notification_emails:
+    - CHANGE-ME@test-only.com
+  bugs:
+    - 123456789
+  data_reviews:
+    - http://test-only.com/path/to/data-review
```

Comment:
Use `example.com` in examples: https://tools.ietf.org/html/rfc2606#section-3

Reply:
Good point.  We have a bunch of other instances of these that I've changed as well.